README.md

```diff
@@ -18,4 +18,34 @@ exponentiations in the contract and verify that the precision obtained is suffic
 The [ligo](ligo) directory contains a [CameLIGO](https://ligolang.org/)
 implementation of a smart-contract.
 
-This code is work in progress and should not be expected to run or even compile.
\ No newline at end of file
+This code is work in progress and should not be expected to run or even compile.
+
+
+## Generating a contract metadata
+
+The contract metadata should be compliant with [TZIP-16](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md).
+
+You can generate the known metadata, but you'll need the `stack` tool installed, see
```

Comment:
Not just here, but in other places there is `known` used, however we should probably not do that.

Aside from some token info (which can be provided) all metadata is known for this contract, so IMO we can remove this.

Reply:
I've remove `known` and make other suggestion according to your feedback. This should be ready for re-review.